README.org

```diff
@@ -424,11 +424,11 @@ install Helm and configure Icomplete to use it as follows:
 The [[https://github.com/raxod502/prescient.el][prescient.el]] library also provides matching of space-separated
 components in any order and it can be used with either the [[https://github.com/raxod502/selectrum][Selectrum]] or
 [[https://github.com/abo-abo/swiper][Ivy]] completion UIs (it does not offer a completion-style that could be
-used with Emacs' default completion UI, Vertico or with Icomplete). The
-components can be matched literally, as regexps, as initialisms or in
-the flex style (called "fuzzy" in prescient). In addition to matching,
-=prescient.el= also supports sorting of candidates (=orderless= leaves that
-up to the candidate source and the completion UI).
+used with Emacs' default completion UI, Mct, Vertico or with Icomplete).
```

Comment:
I'm using mct right now and it's pretty nice!

Reply:
Yes, it is very nice! I can imagine using it myself. But I am also quite accustomed to Vertico (its particular way of interaction and responsiveness). And well there is the Ikea effect. Mct works almost flawlessly for me except a few window resizing/flicker hiccups which I cannot really pin down.

Reply:
I just noticed that consult-preview-mode isn't working for me with consult-imenu (but seems to work just fine with all other consult commands I've tried). This is isn't specific to Mct, it also happens in the completions buffer with default completion. Weird.

Reply:
It could be this: https://github.com/minad/consult/commit/af7f94a24f581e45ae57110b53f27868fd905037. See also https://github.com/oantolin/embark/issues/485. I initially had planned to upstream a `get-completion-candidate` function which could be used on Emacs 28 but then all my upstream contributions stalled.

Reply:
Yep, that was it, thanks. I just hadn't updated consult yet, I guess.